src/Devices/Base/BaseDevice.cs

```diff
@@ -64,5 +68,18 @@ public void SendMessageToCloud (string messageString)
 
         }
 
+        public async Task EnableDevice()
```

Comment:
Should this be a "setDeviceState" type of property that can be set to enabled or dissabled?

Reply:
Done!